git/git.go

```diff
@@ -490,13 +490,45 @@ func (c *gitConfig) IsGitVersionAtLeast(ver string) bool {
 	return IsVersionAtLeast(gitver, ver)
 }
 
+// Prepend Git config instructions to disable Git LFS filter
+func PrependConfigToDisableLFS(args ...string) []string {
```

Comment:
I am not sure I understand what you're saying here.
Would you prefer it if I don't use the additional Git Exec wrapper functions?

In case of config changes disabling LFS wouldn't be safe. But everything else should be, no?